include/zephyr/net/phy.h

```diff
@@ -209,6 +210,10 @@ static inline int phy_configure_link(const struct device *dev, enum phy_link_spe
 {
 	const struct ethphy_driver_api *api = (const struct ethphy_driver_api *)dev->api;
 
+	if (!api->cfg_link) {
```

Comment:
```suggestion
	if (api->cfg_link == NULL) {
```
explicit check for NULL, like in the other apis.